common/src/warp_runner/mod.rs

```diff
@@ -185,6 +187,11 @@ async fn handle_login(notify: Arc<Notify>) {
                             Ok(_id) =>  match wait_for_multipass(&mut warp, notify.clone()).await {
                                 Ok(ident) => match save_tesseract(&warp.tesseract) {
                                     Ok(_) => {
+                                        let pfp = create_user_default_profile_picture(ident.did_key()).unwrap_or_default();
+                                        match warp.multipass.update_identity(IdentityUpdate::Picture(pfp)).await {
+                                            Ok(_) => log::info!("Identity updated successfully with default polkadot placeholder"),
+                                            Err(e) => log::warn!("Failed to update identity with default polkadot placeholder: {}", e),
```

Comment:
It might be best to perform a match on some errors to determine if its safe to actually proceed. Not a blocker though but just a thought on if an error should be returned to the oneshot on say an IO error, etc. Then again, this is for the account creation so this might not be worth returning an error.